libs/extensions/angular/image-banner/src/image-banner.component.scss

```diff
@@ -0,0 +1,171 @@
+@use '@kirbydesign/core/src/scss/utils';
+@use '@kirbydesign/core/src/scss/interaction-state';
+
+$blur-radius: 60px;
+$max-content-width: 324px;
+
+:host(.none) {
+  .blur-image {
+    display: none;
+  }
+
+  .dismiss {
+    // on mobile we always want the button to be white as it sits on top of the image
+    @media (width < 632px) {
```

Comment:
Magic number? Or do we have a breakpoint for this? 🤔

Reply:
There is a breakpoint, forgot to use it!